pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -19,20 +19,33 @@ limitations under the License.
 package cadvisor
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"net/http"
 	"os"
 	"path"
 	"time"
 
+	"github.com/google/cadvisor/accelerators"
 	"github.com/google/cadvisor/cache/memory"
+	"github.com/google/cadvisor/container"
 	cadvisormetrics "github.com/google/cadvisor/container"
+	_ "github.com/google/cadvisor/container/containerd"
```

Comment:
can you split the kubernetes changes into a separate commit than the `hack/update-vendor.sh` commit?

Reply:
ack. on it